codeforces-contests/contest-1265.cpp

```diff
@@ -0,0 +1,69 @@
+//https://codeforces.com/contest/1265/problem/A
+
+#include <bits/stdc++.h>
+
+using namespace std;
+string s;
+const int constant = 294;
+
+int main()
+{
+    #ifndef ONLINE_JUDGE
+    freopen("contest-1265.in", "r", stdin);
+    freopen("contest-1265.out", "w", stdout);
+    #endif
+
+    int testCase;
+    cin >> testCase;
+
+    for (int i = 1; i <= testCase; ++i) {
+        cin >> s;
+        s[s.size()] = '|';
```

Comment:
This is not ok. You are lucky you didnt get Killed by Signal.
If you want to do something like this, you do s += '|' OR s.push_back('|').
That being said you dont need that special character anyway, you can just check if the next index is s.size().